media/css/mozorg/internet-health/hub/_campaign.scss

```diff
@@ -1,133 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-@import '../../../pebbles/includes/lib';
-
-/* -------------------------------------------------------------------------- */
-// Campaign hero image
-
-$color-campaign-background: #0f161b;
-
-.hero-image {
-    background: $color-campaign-background;
-
-    img {
-        display: block;
-        margin: 0 auto;
-    }
-}
-
-// hide the header for english locales that see the campaign hero image
-html[lang^="en"] {
-    .header-intro {
-        display: none;
-    }
-
-    .podcast {
-        padding-top: 60px;
-    }
-}
-
-/* -------------------------------------------------------------------------- */
-// Campaign articles & Download Firefox CTA
-
-.campaign-articles,
-.download-firefox {
-    background: $color-campaign-background;
-
-    .content:before {
-        content: none;
-    }
-
-    .news-feed,
-    .content-container {
-        padding: 20px;
-    }
-
-    @media #{$mq-tablet} {
-        .news-feed,
-        .content-container {
-            padding: 20px 40px;
-        }
-    }
-}
-
-.campaign-articles {
-    .content {
-        padding-bottom: 0;
-    }
-
-    .news-feed {
-        background: #fff;
-    }
-}
-
-/* -------------------------------------------------------------------------- */
-// Download Firefox CTA specific styles
-
-.download-firefox {
-
-    .content {
-        padding-top: 0;
-    }
-
-    .content-container {
-        @include clearfix;
-        background: #dddada url('/media/img/firefox/template/logo-jumbo-high-res.png') top 20px center no-repeat;
```

Comment:
Does not look like anything else uses this image.

Reply:
Yeah I wasn't sure about that one, since it's in the template folder. It was added for a different page and I think just reused here so I decided not to delete it. We'll end up replacing all of these logo images in a few weeks anyway.